spec.html

```diff
@@ -936,6 +989,24 @@ <h1>The AsyncContext Object</h1>
 
     <emu-clause id="sec-asynccontext-abstract-operations">
       <h1>AsyncContext Abstract Operations</h1>
+
+      <emu-clause id="sec-gethostdefinedasynccontextmapping" type="abstract operation">
+        <h1>GetHostDefinedAsyncContextMapping ( ): a List of Async Context Mapping Records</h1>
+        <dl class="header">
+        </dl>
+        <p>An implementation of GetHostDefinedAsyncContextMapping must conform to the following requirements:</p>
+        <ul>
+          <li>It must perform and return a List of Async Context Mapping Records.</li>
```

Comment:
"perform and return" doesn't make sense, grammatically.  It looks like maybe you copied this from HostCallJobCallback's "It must perform and return the result of Call(_jobCallback_.[[Callback]], _V_, _argumentsList_)." but in that case, it's basically defining a tail-call of another abstract operation, so "perform" is referring to the Call operation, and that's not present here.  I think this just wants to be "It must return a List of Async Context Mapping Records."  But at that point you're just reiterating the return type from the signature, so this bit seems unnecessary.  Maybe it would be better to follow the pattern from #sec-hostgetimportmetaproperties?  (unless we add the requirement that it always return the same mapping)